.travis.yml

```diff
@@ -21,14 +18,43 @@ env:
     - GH_REF: github.com/paulgibbs/behat-wordpress-extension
     - secure: RDArf/FuQFEz0O5n9NnWq9efe8tOH8eQIH28rNhSGA1lZ57zo2+3fU6QRvZrvW4fb7bIN6cvZSlOnsO3RYrf/UUwPSItjn9Jiia1wo70iqvdWw9VRtZ7YWysHkxi3tfEhIoBkHJMXJ7xhPrGg/qAevuoXWAkuocRUg370Fpe1oyMT6FC9brCsfgt6kJeCDEjeRrxUVCaozpTpX8/K/Glfu2lU8jiF6U5GvoT8+fzVsAgs/zJz9CoJcAzhWl/b2UrF5H2pqd6LoNad93NM5c5sG69E3VtuxO0ahECk1d2FttL6cEeSXM81ZRO4tRXfUfkvLu43kRVdNSYdRlD+XjQ2s+GcNvVMvmS/kun19rOxxlf+rgbYMcVoENC/DsSu0Qvk/y++POJHNLVwPECAviHK1qEgvdhMMsN87lh/op0aDLB1mKIYj4DbF5gHt4d16QfAJVTx8SeYbqRJrbHHXcRG3HHWwPp5mxWUoknvAkZgWRqZgByXdW1oB81qqqDZ42L1iGCmjufDYF6gw9dzaUERJzO/EmgS6dqeflUJIYxjn8bNrrCwEvYOTFNElHg78M/hUs046i4Ui9xd1kJUhEKduX9HoLalfzTwF+qAz+lEHOGXkIs/1hWB6UhacggnWPj7/GPXi4OgWwPPgJuHJ+MfmO/frXx3h/mYTx8OzMv8NU=
 
-before_script:
+before_install:
+  - sudo apt-get update > /dev/null
+  - composer self-update
+
+  # Install Apache
+  - sudo apt-get install -y --force-yes apache2 libapache2-mod-php5 php5-curl php5-mysql php5-intl
+  - sudo a2enmod rewrite
+  - export WORDPRESS_SITE_DIR=$(pwd)/www/
+  - sudo sed -i -e "s,/var/www,$(pwd)/www,g" /etc/apache2/sites-available/default
+  - sudo sed -i -e "s,AllowOverride None,AllowOverride All,g" /etc/apache2/sites-available/default
+  - sudo sed -i -e "/DocumentRoot/i\ServerName wordpress.dev" /etc/apache2/sites-available/default
+  - echo "127.0.0.1 wordpress.dev" | sudo tee -a /etc/hosts
+  - sudo /etc/init.d/apache2 restart
+
+install:
   - phpenv config-rm xdebug.ini
+
+  # Install dependencies
   - composer validate
-  - composer install --no-progress
+  - composer install --no-interaction --prefer-dist --no-progress;
+
+  # install wordpress
+  - bash ./ci/prepare.sh wordpress root '' localhost $WP_VERSION
```

Comment:
Where is `$WP_VERSION` set?

(p.s. I think we should pick the last stable major branch, rather than its dev-master. I've seen many in-progress changes break stuff in tests in the past.)